07-advanced-git/07-01-git-log/README.md

```diff
@@ -0,0 +1,135 @@
+# Table of Contents
+- [Table of Contents](#table-of-contents)
+- [Git Log](#git-log)
+  - [Project set up so-far](#project-set-up-so-far)
+    - [Basic git log](#basic-git-log)
+    - [Flags](#flags)
+      - [oneline](#oneline)
+      - [graph](#graph)
+      - [all](#all)
+  - [Wrapping up](#wrapping-up)
+    - [Additional Resources to read](#additional-resources-to-read)
+    - [GUI applications to help you visualise the git log](#gui-applications-to-help-you-visualise-the-git-log)
+    - [Github:](#github)
```

Comment:
Added table of contents above for easier navigation on the markdown and website.

Reply:
How is this being generated? I'm concerned because if this is hand generated, if we re-generate the PDF, we have to remove this portion otherwise there will be two table of contents :thinking:

Reply:
I removed when generating the pdf.  
So there is only 1 table in the pdf  

I generated using a vs ccode extension based on the headings.

Reply:
I see, I think it's better if it isn't included since it would be easy to forget that you have the table of contents when exporting to the PDF. 

I might be working on getting the entire guide hosted on a proper documentation site generator that will auto-create the table of content that way. It's in the works but definitely something I'll get settled when I can.

Reply:
Ah alright then, I will remove it.